torch/_dynamo/variables/torch.py

```diff
@@ -160,6 +160,7 @@
     torch.accelerator.is_available,
     torch.cuda.get_device_properties,
     torch.cuda.is_available,
+    torch._C._nn._parse_to,
```

Comment:
I'm not sure if we can constant fold this directly, because one of the argument might be an intermediate tensor...? But then I don't quite see how that tensor is used in the actual implementation: https://github.com/pytorch/pytorch/blob/361692cb544114b7377da8a24c1fbf16d314edc1/tools/autograd/templates/python_nn_functions.cpp#L39-L45

@zou3519 any idea?

Reply:
I have never seen anyone use this before, but this is how you use the third overload:
```py
m = torch.nn.Linear(3, 3).cuda()
cpu_tensor = torch.randn(3, dtype=torch.float64)
m.to(cpu_tensor)  # in-place modifies `m` to be on cpu device and float64 dtype
```